metal/loadbalancers/metallb/metallb.go

```diff
@@ -65,7 +62,7 @@ var (
 )
 
 // func NewLB(k8sclient kubernetes.Interface, k8sApiextensionsClientset *k8sapiextensionsclient.Clientset, config string) *LB {
-func NewLB(k8sclient kubernetes.Interface, config string) *LB {
+func NewLB(k8sclient kubernetes.Interface, config string, useCrdConfiguration bool) *LB {
```

Comment:
If it's important _not_ to change this function signature, and if the `metallb:///<configNamespace>/<configMapName>` format for the `config` string was invented by us, we could alternatively introduce a different config string format for newer MetalLB.